test/general/version.pass.cpp

```diff
@@ -21,19 +21,6 @@
 
 #include "support/utils.h"
 
-#if defined(__has_include)
-#define HAS_EXECUTION_HEADER __has_include(<execution>)
-#else
-#define HAS_EXECUTION_HEADER 0
-#endif
-
-#if __cplusplus < 201703L || !HAS_EXECUTION_HEADER
-static_assert(_PSTL_VERSION == 11000, "");
-static_assert(_PSTL_VERSION_MAJOR == 11, "");
-static_assert(_PSTL_VERSION_MINOR == 00, "");
-static_assert(_PSTL_VERSION_PATCH == 0, "");
```

Comment:
I would keep these checks under proper conditions; it does make sense to ensure that the upstream PSTL version is defined, because some applications might in theory use it.